_includes/setup/path-win.md

```diff
@@ -0,0 +1,23 @@
+### Update your path
```

Comment:
I'd think a user variable is enough. (Isn't the setup we are recommending for Linux/Mac also just setting it for the current user?)